workspace/third/faire/common/some.proto

```diff
@@ -0,0 +1,6 @@
+syntax = "proto3";
+
+package api.v2;
+
+import "faire.common.collection2.proto";
+import "faire.common.collection.proto";
```

Comment:
Imports in protobuf use the file path, not the package name:

```suggestion
import "faire/common/collection.proto";
import "faire/common/collection2.proto";
```

This might take a minute to get used to, if you are coming from languages that do it differently 😊 

With this change, the module builds fine. Note that `buf breaking` will still try to build the previous commit to compare against.